### DISCUSSION TITLE
Ansys electronics 2021 r1 材料设置coreloss

### DISCUSSION THREAD

**Discussion by hh-wu** [POLYU-EMLAB/polyu-emhub#6] [Q&A] [answered]
现在找不到loss type选项，应该如何设置？

**Comment by jiangmy97** [comment, marked as answer]
在View/Edit Material界面中，勾选右方View/Edit Material for>Active Project或All Properties即可。 
![image](https://user-images.githubusercontent.com/41866272/131207103-b23c855f-1596-4e74-84ea-a0c0f432db36.png)
